src/main/java/com/esir/jxc/mpc/Config.java

```diff
@@ -0,0 +1,53 @@
+package com.esir.jxc.mpc;
+
+import org.elasticsearch.client.Client;
+import org.elasticsearch.client.transport.TransportClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.transport.InetSocketTransportAddress;
+import org.elasticsearch.transport.client.PreBuiltTransportClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.elasticsearch.core.ElasticsearchOperations;
+import org.springframework.data.elasticsearch.core.ElasticsearchTemplate;
+import org.springframework.data.elasticsearch.repository.config.EnableElasticsearchRepositories;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+@Configuration
+@EnableElasticsearchRepositories(basePackages = "com.esir.jxc.mpc.repository")
+public class Config {
+
+    @Value("${elasticsearch.host}")
+    private String host;
+
+    @Value("${elasticsearch.port}")
+    private int port;
+
+    @Value("${elasticsearch.clustername:pocket-cluster}")
+    private String clusterName;
+
+    @Value("${elasticsearch.home:D:/Programmes/elasticsearch/5.6.0}")
+    private String home;
+
+    @Bean
+    public Client client() {
+        Client client = null;
+        try {
+            final Settings elasticsearchSettings = Settings.builder()
+                    .put("path.home", host)
+                    .put("cluster.name", clusterName).build();
+            TransportClient transportClient = new PreBuiltTransportClient(elasticsearchSettings);
+            client = transportClient.addTransportAddress(new InetSocketTransportAddress(InetAddress.getByName(host), port));
+        } catch (UnknownHostException e) {
+            e.printStackTrace();
+        }
+        return client;
```

Comment:
```suggestion
        return null;
```
A deplacer dans le catch